<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PipeRefluxWidgetsClass</class>
 <widget class="QMainWindow" name="PipeRefluxWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1660</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PipeRefluxWidgets</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>1291</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>100</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>1181</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chk_1">
        <property name="text">
         <string>1号泵</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_2">
        <property name="text">
         <string>2号泵</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_3">
        <property name="text">
         <string>3号泵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_4">
        <property name="text">
         <string>4号泵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_5">
        <property name="text">
         <string>5号泵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_6">
        <property name="text">
         <string>6号泵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_7">
        <property name="text">
         <string>7号泵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_8">
        <property name="text">
         <string>8号泵</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_9">
        <property name="text">
         <string>9号泵</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>345</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>全选</string>
     </property>
     <property name="btnStyle" stdset="0">
      <string>normalBtnpink</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择泵</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>1331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>————————————————————————————————————————————————————————————————————————————————————————————————————————————————</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>560</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
    <property name="btnStyle" stdset="0">
     <string>normalBtnRed</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_all">
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
